api/compression/index.md

```diff
@@ -30,6 +31,8 @@ Compressed chunks have the following limitations:
     after constraint creation.
 *   [Timescale SkipScan][skipscan] does not currently work on compressed chunks.
 
+<Deprecated2180 /> Replaced by <a href="https://docs.timescale.com/api/latest/hypercore/">Hypercore</a>.
```

Comment:
We may emphasize that compression works in the new version.

Reply:
Done